test/onnx/test_utility_funs.py

```diff
@@ -1457,8 +1457,8 @@ def _test_deduplicate_initializers(self, torchscript=False):
         class MyModule(torch.nn.Module):
             def __init__(self):
                 super().__init__()
-                self.layer1 = torch.nn.Linear(3, 3)
-                self.layer2 = torch.nn.Linear(3, 3)
+                self.layer1 = torch.nn.Linear(256, 256)
```

Comment:
It may also be helpful to point out the chose of size here. maybe having a constant like `LARGE_TENSOR_SIZE = 256`?